Samples/Windowing/cs-wpf/wpf_packaged_app/wpf_packaged_app.csproj

```diff
@@ -10,7 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.1.0-preview2" />
```

Comment:
This should be "1.1.0" not targeting the preview2 release.
Note: this is perhaps not blocking for this PR, but needs to be updated once the 1.1.0 release is live.